src/mudlet.h

```diff
@@ -473,6 +476,7 @@ class mudlet : public QMainWindow, public Ui::main_window
     QSystemTrayIcon mTrayIcon;
     bool mUsingMudletDictionaries = false;
     bool mWindowMinimized = false;
+    MudletServer* mServer;
```

Comment:
Maybe `MudletInstanceCoordinator` would be better.

Reply:
I still think we need to introduce `QtDBus` into Mudlet so that the first instance started sets up a connection (`org.mudlet.DBus`) that other instances detect and latch onto to coordinate between themselves as needed - this may also have a bearing on another issue related to handling `telnet://` URLs...

Reply:
I'm not sure sure, as dbus isn't something that runs on windows or macos natively.

Reply:
According to https://www.freedesktop.org/wiki/Software/dbus/ the Windows port is established and used and available for MSVC and mingw-w32/w64(gcc) and cygwin(gcc). As it can also used between applications without a central deamon (provided I guess that they both know to try/use it) and we are talking abut multiple instances of the same Mudlet application this doesn't sound like a blocking issue - OTOH I don't see any mention of MacOS so am not sure how that OS would be covered...

Reply:
Ah, https://doc.qt.io/qt-6/macos-issues.html#d-bus-and-macos covers this subject.

Reply:
Sure, but it is still not native to the platform. Without any Qt applications, dbus does not run on windows or macos. They have their own, different message buses.